addon/components/o-s-s/infinite-select.stories.js

```diff
@@ -125,7 +125,12 @@ export default {
     }
   },
   parameters: {
-    layout: 'fullscreen'
+    layout: '',
```

Comment:
That's change anything but there are a reason, why in array-input.stories.js you delete the line layout and here layout is empty ?